autoendpoint/Cargo.toml

```diff
@@ -12,9 +12,9 @@ edition = "2018"
 # The `autoendpoint` branch merges these three PRs together.
 # The version of a2 at the time of the fork is v0.5.3.
 a2 = { git = "https://github.com/Mcat12/a2.git", branch = "autoendpoint" }
-actix-web = "2.0"
-actix-rt = "1.0"
-actix-cors = "0.2.0"
+actix-web = "3.1"
```

Comment:
With the Actix update, this might improve sentry error reporting:
https://github.com/mozilla-services/autopush-rs/blob/9b93817776bb7c8f27383185a1b5ce08607a6b35/autoendpoint/src/middleware/sentry.rs#L83-L84

Reply:
Heh, indeed. Just had a bit of an issue with a PR that was more Katamari Damacy than I'd prefer, so trying to keep future PRs a lot more manageable. 

Filed https://github.com/mozilla-services/autopush-rs/issues/228